tensorflow/lite/kernels/internal/types.h

```diff
@@ -1081,11 +1081,16 @@ struct UnpackParams {
 
 struct LeakyReluParams {
   float alpha;
+};
+
+
+struct LeakyReluParamsQuant {
```

Comment:
normally we don't create a new params but just put it together with the float

Reply:
Merged to float Params.